test.py

```diff
@@ -208,11 +208,13 @@ def main():
     # Run example tests
     if flags['example'] in args.suites:
         run_example_tests()
-        validate_example_nwbs()
+        validate_nwbs()
 
     # Run integration tests
     if flags['integration'] in args.suites:
+        os.environ['CLEAN_NWB'] = 'False'
```

Comment:
Is there an easy way to remove all of the created files after validation is completed?

Reply:
Perhaps add a parameter to `validate_nwbs` to set `delete_files` = True/False and then within the loop over files, check `delete_files` and delete the file if true.

Reply:
Or maybe the validation should not happen here but in the integration tests itself and we can remove the `CLEAN_NWB` fiddling here?